### ISSUE TITLE
[Bug] meta-data does not show up in frontend via the folder option

### ISSUE THREAD

**Issue by medarob** [Hauer-Heinrich/hh_slider#12]
Hi,

I wanted to output some videos with the folder option and wanted to add the meta-data (title, description, caption) which I added to the file directly (filelist). I wanted to place the meta-data information under the video. The problem is that the meta-data does not show up in the frontend.

```
<f:if condition="{data.tx_hhslider_content_type} == 3">
            <f:if condition="{folderFiles}">
                <f:for each="{folderFiles}" as="file" iteration="i">
                    <f:if condition="{file.extension} == 'mp4' || {file.extension} == 'ogv' || {file.extension} == 'ogg'">
                        <f:then>
                            <div>
                            <f:media class="video-embed-item" file="{file}" alt="{file.alternative}" title="{file.title}" additionalConfig="{settings.media.additionalConfig}" />
                             <p>Erweiterung: {file.extension}<br/>Title: {file.title}<br/>Description: {file.description}<br />Caption: {file.caption}</p>
                            </div>
                        </f:then>
```

The output looks like this:
![grafik](https://user-images.githubusercontent.com/33951787/82795000-aa12ae00-9e73-11ea-8bec-b8e7cc0359bc.png)

Only `{file.extension}` seems to work.

I use TYPO3 9.5.15 and hhslider 0.1.62.

**Comment by medarob** [comment]
It seems like that you need to use {files.**properties**.title}, {file.**properties**.description} to make it work for files (option 3: slider via folder files).
I found this issue https://forge.typo3.org/issues/54056 with the solution.

So, the attribute in the `<f:media>`-Tag has to be changed from `title="{file.title}"` to `title="{file.properties.title}"`. Not sure if `alt="{file.alternative}"` is available in that context, too. Maybe it can be changed to `alt="{file.properties.caption}"`?

**Comment by Teisi** [comment]
what would you like the hh_slider to do here?

**Comment by medarob** [comment]
The initial idea was to create a video slider which shows me 4 videos at once and beneath each video file the title and description from the metadata. Therefore I used the CE option, created multiple Text&Media elements and included in each element 1 video.

That works in all modern browsers but not in IE11.

In IE11 it looked like this (top):

![grafik](https://user-images.githubusercontent.com/33951787/82817672-b0b61b00-9e9d-11ea-90c2-304575afcb10.png)

and I don't know what causes this problem but I noticed that if I use the folder option it works (bottom). I think it's because of the multiple tags that were created when I use the text&media elements. 

![grafik](https://user-images.githubusercontent.com/33951787/82814502-96793e80-9e97-11ea-84fe-c40d5a556136.png)

This approach works in IE11 and the slider shows up fine. The only problem now is that I get the  following message:
![grafik](https://user-images.githubusercontent.com/33951787/82815363-4ef3b200-9e99-11ea-9e81-b8447d56b088.png)

The same video works on another page in IE11.

If I use the CE option with the Text&Media CE and a videofile the video plays also fine in IE11 but the height is wrong and not sure how to fix that.

But this issue was about that {file.description} and {file.title} are not working with the folder option because there are no getters in `TYPO3\CMS\Core\Resource\File` and therefore `{file.properties.title}`  and maybe `{file.properties.alternative}` has to be used here:
```
<f:media class="video-embed-item" file="{file}" alt="{file.alternative}" title="{file.title}" additionalConfig="{settings.media.additionalConfig}" />
```
